service/key/key.go

```diff
@@ -298,3 +299,18 @@ func IngressClassName() string {
 func OpsgenieKey() string {
 	return "opsgenie.key"
 }
+
+// PrometheusDiskSize returns the desired disk size based on the
+// value of annotation monitoring.giantswarm.io/prometheus-disk-size
+func PrometheusDiskSize(value, defaulVal string) string {
+	switch value {
+	case "small":
+		return "30Gi"
+	case "medium":
+		return defaulVal
```

Comment:
I don't much like that `small` and `large` values are defined there, but not `medium`.
Couldn't we have all `label=size` definitions in one place?

Reply:
I agree. I would create a struct passed to the config instead of adding another function in key.
Maybe something like PrometheusSize.

It would have a New method that would set medium to config.PrometheusSize

Reply:
I didn't' fully understand